### PR TITLE
so/manifesto page

### LINKED ISSUE #4: Create page: Manifesto

### PR DESCRIPTION
**Pull request by sssoja**
### What changes have you made?

- created manifesto page with client approved responsive styles

### Which issue(s) does this PR fix?

Fixes #4 

### Screenshots (if there are design changes)

Screenshot 2020-07-16 at 15 37 48
Screenshot 2020-07-12 at 20 49 43
Screenshot 2020-07-16 at 15 39 02

### How to test
- `yarn start` and add /manifesto to the `localhost:3000` url 
- use dev tools to test mobile styles

### CODE REVIEW

File: src/components/Manifesto.tsx [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+          ]}
+        >
+          <Paragraph
+            color="athensGray"
+            fontSize={[1, 3, 4]}
+            p={3}
+            textAlign="justify"
+            m={5}
+          >
+            {t("manifesto.manifesto")}{" "}
```
Comment (mantagen) [inline src/components/Manifesto.tsx:96]: No need for `{" "}` at the end i don't think? Same for line 105

**Comment by mantagen** [comment]
image


Not sure what's happening here. I feel like it should just be one column on mobile?

File: src/components/Manifesto.tsx (hunk 1 above)
Comment (sssoja) [reply src/components/Manifesto.tsx:96]: Weird lint setting. Likes to sneak them in automatically! Will get rid

**Comment by sssoja** [comment]
iPhone SE (and other smaller screens) fixed.

Screenshot 2020-07-13 at 16 56 03

**Pull request merged**